### DISCUSSION TITLE
switching pages in the breadcrumbs

### DISCUSSION THREAD

**Discussion by marektonik** [twilio-labs/paste#3356] [Q&A] [answered]
hey, we are trying to reproduce patter that is being used in Evergreen. It's a breadcrumb combined with minimal button which - once we click - opens a list and we can jump into the another page. 

Any suggestions how to handle this type of case?

<img width="500" alt="Zrzut ekranu 2023-07-27 o 09 11 40" src="https://github.com/twilio-labs/paste/assets/140410476/b36650f9-f6a5-41f2-8b8a-9eb9f4e153a1">

**Comment by loreina** [comment, marked as answer]
Hi @marektonik,

You can achieve that composition with our [Breadcrumb](https://paste.twilio.design/components/breadcrumb) and [Menu](https://paste.twilio.design/components/menu#choosing-a-menu-trigger) components. Here's a [CodeSandbox](https://codesandbox.io/s/breadcrumbs-with-actions-tj29b7?file=/src/index.tsx) on how it can composed in code.

While it's possible to recreate this pattern with Paste components, it might be helpful to consider: how widespread is this pattern across the product? How many users rely on the breadcrumb menu? Is it the best way for users to navigate to other pages in the app?

The team would be curious to the answers of those and if there's an opportunity here for an alternative design.

**Reply by ceyerkes** [reply]
From @kevinbtu "I don't have actual click stats, but in terms of how widespread this pattern is in our app - it's pretty widespread. Breadcrumbs is used in the Connections, Unify, and Engage sections of our app (which is the majority). It does allow for a much quicker switching between different sources and destinations. I will try to pull some click stats"
